quixstreams/kafka/admin.py

```diff
@@ -0,0 +1,191 @@
+import logging
+import pprint
+import time
+from asyncio import Future
+from typing import List, Dict, Mapping, Optional
+
+from confluent_kafka.admin import (
+    AdminClient,
+    ConfigResource,
+    KafkaException,  # type: ignore
+    NewTopic,  # type: ignore
+    TopicMetadata,
+)
+
+from quixstreams.models.topics import TopicConfig, TopicList
+
+logger = logging.getLogger(__name__)
+
+__all__ = ("Admin",)
+
+
+def convert_topic_list(topics: TopicList) -> List[NewTopic]:
```

Comment:
Does it make sense to import NewTopic with 
`import NewTopic as ConfluentNewTopic` or something similar, in my opinion it makes the code more readable but its really just an opinion, feel free to simply ignore this

Reply:
Nah I agree with you, I basically did a similar thing later on with Confluent's `TopicPartition`, renaming it `ConfluentPartition`.

I think I'll just rename it `ConfluentTopic` and similarly `TopicMetadata` as `ConfluentTopicMetadata`. Lemme know what you think